packages/core/__tests__/config/load-config.test.ts

```diff
@@ -51,4 +52,49 @@ describe('Config: loadConfig', () => {
     expect(config.environment.getConfiguredTemplateTags()).toEqual({ test: {} });
     expect(config.checkStandaloneTemplates).toBe(false);
   });
+
+  test('locates config in package', () => {
+    const directory = `${testDir}/package-glint-config`;
+    const nodeModulePackageDir = `${directory}/node_modules/@package1`;
+
+    vi.spyOn(require, 'resolve').mockImplementation((id: string | undefined) => {
+      if (id === '@package1/tsconfig.json') {
+        return id.replace('@package1', nodeModulePackageDir);
+      }
+      throw Error(`Cannot resolve module ${id}`);
+    });
+
+    fs.mkdirSync(nodeModulePackageDir, { recursive: true });
+    fs.writeFileSync(
+      `${nodeModulePackageDir}/package.json`,
+      JSON.stringify({
+        name: '@package1',
+        version: '1.0.0',
+      })
+    );
+    fs.writeFileSync(
+      `${nodeModulePackageDir}/tsconfig.json`,
+      JSON.stringify({
+        glint: {
+          environment: 'kaboom',
+          checkStandaloneTemplates: false,
+        },
+      })
+    );
+    fs.writeFileSync(
+      `${directory}/tsconfig.json`,
+      JSON.stringify({
+        extends: '@package1/tsconfig.json',
+        glint: {
+          environment: '../local-env',
```

Comment:
what's local-env?

Reply:
To be honest, I simply followed the example from the previous test which uses a predefined environment in the [beforeEach](https://github.com/typed-ember/glint/blob/53f68241d6f4224fceb8b318f2566e5eb9c9a703/packages/core/__tests__/config/load-config.test.ts#L14)